app/browser/api/ledger.js

```diff
@@ -2373,6 +2373,8 @@ const checkBtcBatMigrated = (state, status) => {
   if (!isNewInstall && !hasUpgradedWallet) {
     state = migrationState.setTransitionStatus(state, true)
     module.exports.transitionWalletToBat()
+  } else {
+    state = migrationState.setTransitionStatus(state, false)
```

Comment:
This is the big bug fix- if it's an existing install and hasn't upgraded yet, it needs to start (or in the case where you quit, restart) the transition process.  Otherwise, it should hide the transition because that is no longer valid

Reply:
Just tested out PR and transition is not triggered for me when I upgrade from 0.18 wallet.